BPRBE/Models/Persistence/ArchitecturalComponent.cs

```diff
@@ -0,0 +1,9 @@
+﻿
+namespace BPRBE.Models.Persistence;
+
+public class ArchitecturalComponent
+{
+    public int Id { get; set; }
```

Comment:
No [BsonId] attribute here?

Reply:
No, because when you assign dependencies between components, you would like to know the id of the component to send it in the dependencies list